<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrixCalc</class>
 <widget class="QMainWindow" name="matrixCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonGauss">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Метод Гаусса</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>711</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите расширенную матрицу системы линейных алгебраических уравений</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>831</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="answerBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>821</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTridiagonal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>530</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Метод прогонки для 
 трехдиагональной матрицы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>720</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Вектор невязки</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
